custom_components/petlibro/switch.py

```diff
@@ -54,6 +54,12 @@ class PetLibroSwitchEntityDescription(SwitchEntityDescription, PetLibroEntityDes
     OneRFIDSmartFeeder: [
     ],
     PolarWetFoodFeeder: [
+        PetLibroSwitchEntityDescription[PolarWetFoodFeeder](
```

Comment:
Could you replace the switch with a button instead?

Initially, the other buttons for the listed devices were switches. However, switches in Home Assistant often don’t display or update correctly. Converting this to a button will help prevent those issues from recurring.

Reply:
Initially I made it as a button, but I found that a switch works really well for this instead. I did hit the issue of the switch not showing the correct state, but changing `@cached_property` to `@property` within `switch.py` seemingly fixed it. I played around with it quite a bit and the switch reliably shows the correct state, with the only exception being when Petlibro's API continues to return a `manualFeedId` from `wetListV3` despite the food door being closed (their official app will display a 'Close door' button despite the door already being closed). This only seems to happen if I'm spamming open/close door events.

Of course, I can still change it to a button if you'd like. If we do decide to not use a switch, would you want a single button for open and close, or separate open and close buttons?

Reply:
So, is your manual feed switch more like an "Open/Close Lid" switch?